src/ui/public/notify/notifier.js

```diff
@@ -316,6 +316,7 @@ Notifier.prototype._showFatal = function (err) {
 };
 
 const overrideableOptions = ['lifetime', 'icon'];
+const defaultTruncationLength = 250;
```

Comment:
I may be reading this module's code incorrectly but it seems that other defaults are stored in [`Notifier.config`](https://github.com/tsullivan/kibana/blob/666f960c70c698b8c34165cf243f606896723a32/src/ui/public/notify/notifier.js#L185). Does it make sense to move the default truncation length there as well?